lapps.vocabulary

```diff
@@ -195,6 +214,45 @@ NamedEntity {
     }
 }
 
+Morphology {
```

Comment:
Would it be useful to sync this with http://universaldependencies.org/u/feat/index.html ?